src/Nethermind/Nethermind.JsonRpc/JsonRpcService.cs

```diff
@@ -118,6 +119,19 @@ private async Task<JsonRpcResponse> ExecuteRequestAsync(JsonRpcRequest rpcReques
         {
             ParameterInfo[] expectedParameters = method.Info.GetParameters();
             string[] providedParameters = request.Params ?? Array.Empty<string>();
+            if (methodName == "trace_callMany")
```

Comment:
we don't want to check methodName in the main JsonRpc interface

Reply:
code smell, if you need to change deserialization logic we need to do it by providing JsonConverter

Reply:
Added converter